shared/version/api.go

```diff
@@ -394,6 +394,7 @@ var APIExtensions = []string{
 	"server_version_lts",
 	"oidc_groups_claim",
 	"loki_config_instance",
+	"instances_efi_vars",
```

Comment:
This commit also needs to include an addition to the https://github.com/canonical/lxd/blob/main/doc/api-extensions.md file at the bottom explaining what the extension provides.

Reply:
Have done. Thanks!